helicopyter.py

```diff
@@ -7,11 +7,17 @@
 from subprocess import check_output
 from typing import Any, TypeVar
 
+from cdk8s import Chart
 from cdktf import App, TerraformElement, TerraformStack
 from constructs import Construct, Node
 from tap import Tap
 
 
+class HeliChart(Chart):
+    def __init__(self, cona: str) -> None:
+        super().__init__(App(outdir='.'), cona)
```

Comment:
Imo this will give us relative pathing issues, depending on where one invokes `python`

Can you do something like `pathlib.Path(__file__).parent`

Reply:
I might not need to specify it. It was mindlessly copied over from cdktf, where it was specified to stop the library from creating an unused directory.